dev/tasks/python-wheels/github.linux.amd64.yml

```diff
@@ -95,7 +87,18 @@ jobs:
         env:
           CROSSBOW_GITHUB_TOKEN: {{ '${{ secrets.CROSSBOW_GITHUB_TOKEN }}' }}
 
+      # TODO(kszucs): only push from master branch
+      - name: Push the docker image
+        shell: bash
+        run: archery docker push python-wheel-manylinux-{{ manylinux_version }}
```

Comment:
Can we remove this step before we merge this?

Reply:
Removed.